cmd/ltctl/loadtest.go

```diff
@@ -68,7 +73,75 @@ func RunLoadTestStartCmdF(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return fmt.Errorf("failed to create terraform engine: %w", err)
 	}
-	return t.StartCoordinator(nil)
+
+	status, err := t.DBStatus()
+	if err != nil {
+		return fmt.Errorf("failed to get DB status: %w", err)
+	}
+
+	if status == dbStopped {
+		if err := t.StartDB(); err != nil {
+			return err
+		}
+
+		fmt.Println("=====================")
+		fmt.Println("Looping until the DB is fully available. You can cancel the command and start the test after some time, or don't do anything and it will automatically start the test after the DB is ready")
+		fmt.Println("=====================")
+		// Now we loop until the DB is available.
+
+		for {
+			status, err := t.DBStatus()
+			if err != nil {
+				return fmt.Errorf("failed to get DB status: %w", err)
+			}
+			if status == dbAvailable {
+				break
+			}
+			fmt.Println("Sleeping... ")
+			time.Sleep(30 * time.Second)
```

Comment:
Isn't 30 second too much for the ping interval?

Reply:
Not really. These take a long time to stop or start. In the order of ~5 minutes.